components/camel-bean-validator/pom.xml

```diff
@@ -34,11 +34,12 @@
 
     <properties>
         <camel.osgi.export>
-            org.apache.camel*;version=${camel-version}
+            org.apache.camel.component.bean.validator*;version=${camel-version}
         </camel.osgi.export>
         <camel.osgi.import>
-            *
+            !org.apache.camel.component.bean.validator,*
```

Comment:
Warning all camel related imports, are managed by `camel-osgi-camel-import`, this cannot work

Reply:
Check https://github.com/apache/camel-karaf/blob/main/pom.xml#L689C43-L689C66 for more details

Reply:
Ok,,I moved it to camel-osgi-camel-import
This  `!org.apache.camel.component.bean.validator,` is to avoid to have it in the import-packages section of the Manifest, which  lead sometimes to exception like `OsgiValidationProviderResolverFactory cannot be cast to ValidationProviderResolverFactory`